types/index.d.ts

```diff
@@ -766,6 +767,25 @@ declare module 'mongoose' {
                     : BufferToBinary<T[K]>;
           } : T;
 
+    /**
+    * Converts any Buffer properties into { type: 'buffer', data: [1, 2, 3] } format for JSON serialization
```

Comment:
```suggestion
    * Converts any Buffer properties into "{ type: 'buffer', data: [1, 2, 3] }" format for JSON serialization
```
(or basically any other delimiter)